package-support-policy.md

```diff
@@ -0,0 +1,292 @@
+| System.Threading.Tasks.Dataflow                              | Upgrade          | Inbox in latest                                     | Inbox in latest                                     |
+| System.Threading.Tasks.Extensions                            | Remove           | Migrated                                            | Migrated                                            |
+| System.Threading.Tasks.Parallel                              | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Threading.Thread                                      | Remove           | Façade                                              | Façade                                              |
+| System.Threading.ThreadPool                                  | Remove           | Façade                                              | Façade                                              |
+| System.Threading.Timer                                       | Remove           | Façade                                              | Façade                                              |
+| System.ValueTuple                                            | Remove           | .NET Framework implementation with in-box façade    | .NET Framework implementation with in-box façade    |
+| System.Windows.Extensions                                    | Upgrade          | Inbox in latest                                     | Inbox in latest                                     |
+| System.Xml.ReaderWriter                                      | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Xml.XDocument                                         | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Xml.XmlDocument                                       | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Xml.XmlSerializer                                     | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Xml.XPath                                             | Remove           | .NET Standard 2.0 implementation, not used anywhere | .NET Standard 2.0 implementation, not used anywhere |
+| System.Xml.XPath.XDocument                                   | Remove           | Simple implementation                               | Simple implementation                               |
+| System.Xml.XPath.XmlDocument                                 | Remove           | Migrated                                            | Migrated                                            |
```

Comment:
@carlossanlop I don't like that we are adding all packages to this document. I think it makes sense documenting the packages that we brought over or intend to bring over but I wouldn't mention any of our actively developed packages like Microsoft.Extensions.*